From 7c4660a3c74ccd1e4dbb784dea7b6fef2bed76ab Mon Sep 17 00:00:00 2001
From: Bruce Leidl <bruce@subgraph.com>
Date: Thu, 4 Apr 2019 14:40:42 -0400
Subject: [PATCH] do not build calendar server

---
 meson.build     | 4 ++--
 src/meson.build | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/meson.build b/meson.build
index e42611e..dae2510 100644
--- a/meson.build
+++ b/meson.build
@@ -73,8 +73,8 @@ else
 endif
 
 atk_bridge_dep = dependency('atk-bridge-2.0')
-ecal_dep = dependency('libecal-1.2', version: ecal_req)
-eds_dep = dependency('libedataserver-1.2', version: eds_req)
+#ecal_dep = dependency('libecal-1.2', version: ecal_req)
+#eds_dep = dependency('libedataserver-1.2', version: eds_req)
 gcr_dep = dependency('gcr-base-3', version: gcr_req)
 gdk_x11_dep = dependency('gdk-x11-3.0')
 gdk_pixbuf_dep = dependency('gdk-pixbuf-2.0')
diff --git a/src/meson.build b/src/meson.build
index 97a5a79..98371a1 100644
--- a/src/meson.build
+++ b/src/meson.build
@@ -1,7 +1,7 @@
 service_data = configuration_data()
 service_data.set('libexecdir', libexecdir)
 
-subdir('calendar-server')
+#subdir('calendar-server')
 subdir('hotplug-sniffer')
 subdir('st')
 subdir('tray')
