From 3ec508d72b45e65b83610680e4fe31fadb13a8d3 Mon Sep 17 00:00:00 2001
From: Bruce Leidl <bruce@subgraph.com>
Date: Thu, 4 Apr 2019 12:39:12 -0400
Subject: [PATCH] Use exe wrapper for custom targets

---
 mesonbuild/backend/backends.py | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/mesonbuild/backend/backends.py b/mesonbuild/backend/backends.py
index 4b9bcb5..f8b8168 100644
--- a/mesonbuild/backend/backends.py
+++ b/mesonbuild/backend/backends.py
@@ -745,6 +745,9 @@ class Backend:
                     and no exe wrapper is defined or needs_exe_wrapper is true.
                     You might want to set it to native instead.'''.format(exe.name))
                 raise MesonException(s)
+            else:
+                exe_path = os.path.join(self.environment.get_build_dir(), self.get_target_filename(exe))
+                return [self.environment.exe_wrapper.get_path(), exe_path]
         if isinstance(exe, build.BuildTarget):
             exe_arr = [os.path.join(self.environment.get_build_dir(), self.get_target_filename(exe))]
         else:
