From 2f0d0585ffc7640e3e10a7260991b33a0783bbd4 Mon Sep 17 00:00:00 2001
From: Bruce Leidl <bruce@subgraph.com>
Date: Thu, 4 Apr 2019 15:34:56 -0400
Subject: [PATCH] new fix cannot find link

---
 build/moz.configure/checks.configure | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/moz.configure/checks.configure b/build/moz.configure/checks.configure
index dc140a42..516652da 100644
--- a/build/moz.configure/checks.configure
+++ b/build/moz.configure/checks.configure
@@ -133,7 +133,7 @@ def check_prog(var, progs, what=None, input=None, allow_missing=False,
 
         for prog in value or progs:
             log.debug('%s: Trying %s', var.lower(), quote(prog))
-            result = find_program(prog, paths)
+            result = find_program(prog.split()[0], paths)
             if result:
                 return result
 
