From 11f025c3fdf32b33ad5ac263b1e638a33f01149b Mon Sep 17 00:00:00 2001
From: Bruce Leidl <bruce@subgraph.com>
Date: Sat, 6 Apr 2019 19:55:22 -0400
Subject: [PATCH] don't run power module tests

---
 plugins/power/meson.build | 43 ---------------------------------------
 1 file changed, 43 deletions(-)

diff --git a/plugins/power/meson.build b/plugins/power/meson.build
index 8babf97..0e0d506 100644
--- a/plugins/power/meson.build
+++ b/plugins/power/meson.build
@@ -102,48 +102,5 @@ if host_is_linux
   )
 endif
 
-output = 'gsdpowerconstants.py'
-
-gsdpowerconstants_py = custom_target(
-  output,
-  input: 'gsd-power-constants.h',
-  output: output,
-  build_by_default: true,
-  command: [join_paths(meson.current_source_dir(), 'gsd-power-constants-update.pl'), '@INPUT@', '@OUTPUT@']
-)
-
-output = 'gsdpowerenums.py'
-
-gsdpowerenums_py = custom_target(
-  output,
-  output: output,
-  capture: true,
-  build_by_default: true,
-  command: [gsd_power_enums_update]
-)
-
-test_py = find_program('test.py')
 
-envs = environment()
-#envs.prepend('G_DEBUG', 'fatal-warnings')
-envs.set('BUILDDIR', meson.current_build_dir())
-envs.set('TOP_BUILDDIR', meson.build_root())
-envs.set('LD_PRELOAD', 'libumockdev-preload.so.0')
-envs.set('NO_AT_BRIDGE', '1')
-envs.set('HAVE_SYSFS_BACKLIGHT', host_is_linux ? '1' : '0')
-
-if get_option('b_sanitize').split(',').contains('address')
-  # libasan needs to be loaded first; so we need to explicitly preload it
-  envs.set('POWER_LD_PRELOAD', 'libasan.so.5')
-endif
-
-foreach i : [ 1, 2, 3, 4, 5, 6, 7, 8 ]
-  test(
-    'test-power @0@/8'.format(i),
-    test_py,
-    args: [ 'PowerPluginTest@0@'.format(i) ],
-    env: envs,
-    timeout: 100
-  )
-endforeach
 
