From e03f5efe9ac52ec06257ec8cf9ef45eb109d0f43 Mon Sep 17 00:00:00 2001
From: Bruce Leidl <bruce@subgraph.com>
Date: Sat, 6 Apr 2019 20:23:15 -0400
Subject: [PATCH] remove private get_client_pid

---
 src/core/window-private.h | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/core/window-private.h b/src/core/window-private.h
index 9fe5bc5..c1a0e9e 100644
--- a/src/core/window-private.h
+++ b/src/core/window-private.h
@@ -771,8 +771,6 @@ void meta_window_handle_leave (MetaWindow  *window);
 void meta_window_handle_ungrabbed_event (MetaWindow         *window,
                                          const ClutterEvent *event);
 
-uint32_t meta_window_get_client_pid (MetaWindow *window);
-
 void meta_window_get_client_area_rect (const MetaWindow      *window,
                                        cairo_rectangle_int_t *rect);
 void meta_window_get_titlebar_rect (MetaWindow    *window,
