From 4cf17283901981e0528ef889fa610bef79382dc1 Mon Sep 17 00:00:00 2001
From: Bruce Leidl <bruce@subgraph.com>
Date: Sat, 6 Apr 2019 20:10:23 -0400
Subject: [PATCH] export get_client_pid

---
 src/meta/window.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/meta/window.h b/src/meta/window.h
index 6261fff..2b5f711 100644
--- a/src/meta/window.h
+++ b/src/meta/window.h
@@ -334,6 +334,9 @@ guint32     meta_window_get_user_time (MetaWindow *window);
 META_EXPORT
 int         meta_window_get_pid (MetaWindow *window);
 
+META_EXPORT
+guint32     meta_window_get_client_pid (MetaWindow *window);
+
 META_EXPORT
 const char *meta_window_get_client_machine (MetaWindow *window);
 
